gnd/src/main/java/com/google/android/gnd/persistence/sync/LocalMutationSyncWorker.java

```diff
@@ -72,10 +78,14 @@ public Result doWork() {
     ImmutableList<Mutation> mutations = localDataStore.getPendingMutations(featureId).blockingGet();
     try {
       Timber.v("Mutations: %s", mutations);
-      processMutations(mutations).blockingAwait();
+      processMutations(mutations)
+          .doOnSubscribe(__ -> sendNotification(UploadProgress.starting()))
```

Comment:
Should we encapsulate all this setup in a method since it's pretty much the same for all workers? Might be overkill since we only have three workers--just a thought.

Reply:
dbf0ce9 PTAL